src/main/kotlin/racingcar/domain/RaceManager.kt

```diff
@@ -0,0 +1,28 @@
+package racingcar.domain
+
+class RaceManager(
+    private val moveCount: Int,
+    private var cars: List<Car>,
+) {
+    var raceHistories: List<RaceHistory> = emptyList()
+        private set
+
+    fun startRacing() {
+        repeat(moveCount) {
+            cars = getMovedCars(cars)
+
+            raceHistories = raceHistories + RaceHistory(cars)
+        }
+    }
+
+    fun getRacingChampions(): List<String> {
+        val maxPosition = cars.maxByOrNull { it.position }?.position
+
+        return cars.filter { it.position == maxPosition }.map { it.name }
```

Comment:
[객체에게 메시지를 보내](https://tecoble.techcourse.co.kr/post/2020-04-28-ask-instead-of-getter/)보면 어떨까요? 😃

Reply:
추가적으로 컬렉션 확장 함수들을 체이닝으로 호출하는 경우에 대해서는 
한 줄에 하나의 메서드를 호출하도록 수정해주시면 조금 더 코드의 가독성이 높아진다고 생각해요. 😃

```suggestion
        return cars.filter { it.position == maxPosition }
            .map { it.name }
```

이렇게 했을 때의 장점은 아래와 같이 호출하는 컬렉션 확장함수가 많은 경우에는 IDE에서도 각 함수 호출에 대한 결과 타입을 알기쉽게 보여줘서 코드 작성시에도 도움을 받을 수 있어요.

<img width="541" alt="image" src="https://github.com/user-attachments/assets/69bf672d-f0ee-4179-af92-283c21525013">

Reply:
알려주신 링크 페이지 참고해서 개선해보도록 하겠습니다!